### ISSUE TITLE
Accepted filetypes parameter in FileDropper component does not support file extensions

### ISSUE THREAD

**Issue by jurriaanpro** [holoviz/panel#7373]
#### ALL software version info
<details>
    <summary>Software Version Info</summary>

```plaintext
panel v1.5.2
python 3.11.8

FilePond 4.31.4 is retrieved when loading component (not sure if relevant)
```
</details>

#### Description of expected behavior and the observed behavior
Panel docs state that it should be possible to specify file extension in the `accepted_filetypes` parameter of the FileDropper component. However, when specifying `.txt` and dropping a file `example.txt`, the error as seen in the screenshot below shows.

The underlying FilePond library seems to be only supporting mime types. See https://pqina.nl/filepond/docs/api/plugins/file-validate-type/#properties

I guess that either Panel docs is invalid or that code needs to be updated to transform specified extensions into MIME types.

#### Complete, minimal, self-contained example code that reproduces the issue

```python
import panel as pn

file_dropper = pn.widgets.FileDropper(accepted_filetypes=[".txt"])

file_dropper
```
#### Screenshots or screencasts of the bug in action
![Screenshot 2024-10-08 at 11 36 19](https://github.com/user-attachments/assets/6f6d4d76-85a9-4cfa-8c60-7395122d6d10)

- [x] I may be interested in making a pull request to address this in case its a documentation issue.

**Comment by ahuang11** [comment]
Thanks for reporting this! I believe this is a duplicate of https://github.com/holoviz/panel/issues/7153

**Comment by jurriaanpro** [comment]
Closed as duplicate of #7153
